### ISSUE TITLE
user error and framework message- runtime configuration is missing

### ISSUE THREAD

**Issue by aradhakrishnanGFDL** [NOAA-GFDL/MDTF-diagnostics#676]
When the runtime configuration does not exist:

Expected behavior: An error message to check the validity of the config file passed with -f. 
Current behavior: The framework exits, without an error/warning. 

```
same behaviour with: mdtf, mdtf_framework 
(_MDTF_base) a1r:/home/a1r/MDTF-diagnostics> ./mdtf_framework.py -f runtime_config_am5test_wheeler.jsonc

(_MDTF_base) a1r:/home/a1r/MDTF-diagnostics> 
```

When the path is valid, it does pick it up.
